include/glow/Importer/Caffe2ModelLoader.h

```diff
@@ -91,11 +96,14 @@ class Caffe2ModelLoader
   /// outputs with specific names and types.
   /// If \p errPtr is not null then if an error occurs it will get assigned
   /// there otherwise if an error occurs it will abort.
+  /// \p bindings is an optional parameters. If it is provided, then trainable
+  /// weights will be added to the graph as Placeholders.
```

Comment:
Please change this comment to say that the loaded weights will be discarded, and are replaced by trainable weights that are initialized for training.